.editorconfig

```diff
@@ -14,3 +14,6 @@ trim_trailing_whitespace = true
 max_line_length = off
 insert_final_newline = false
 trim_trailing_whitespace = false
+
+[*.html]
+ij_html_quote_style = double
```

Comment:
Intellij reads Prettier fairly. Our prettier config says that we want to use single quote (everywhere). But in HTML, it looks awkward. Unfortunately, there is no standard way to indicate to use double quote for HTML only with Editorconfig (https://github.com/editorconfig/editorconfig/issues/410) and Prettier.

Reply:
Does it make sense to have the editorconfig at all if we have prettier?

Reply:
When I format with Prettier through Intellij, I got single quote on HTML (with a default, install). Prettier doesn't allow specifying double quote for HTML. I'm not sure why vscode let us use double quote. I think it's built-in. I'm able to specify the quote type it on my Intellij Config, but as you say, it would make sense to be ready for any developer who would like to do a PR here.

Reply:
We could also just use the default (https://prettier.io/docs/en/options#quotes) which is based on the number of escapes necessary. I don't have a personal preference other than reduce complexity.

Reply:
@imagoiq would an override for html files only work?
https://prettier.io/docs/en/configuration.html#configuration-overrides